tslint.json

```diff
@@ -29,11 +30,11 @@
             "static-before-instance",
             "variables-before-functions"
         ],
-        "no-any": true,
-        "no-arg": true,
+        "no-any": false,
+        "no-arg": false,
         "no-bitwise": true,
         "no-console": [
-            true,
+            false,
```

Comment:
+1, I did the same locally. Maybe we can have different rules for dev builds and prod builds?

Reply:
Exactly, I think we do need dev versus prod tslint rules.
That should be a separate task. Probably when we automate CI/CD.